src/components/multi_selection_list/multiselection_list.js

```diff
@@ -217,30 +215,10 @@ class MultiSelectionList extends PureComponent {
   }
 
   listFilter() {
-    const { searchPlaceholder, searchIcon } = this.props;
-
-    return (
-      <FormControl
-        className={classNames(
-          styles.list_filter_container,
-          styles.search_wrapper
-        )}
-        fullWidth
-      >
-        <Input
-          id="search"
-          value={this.state.searchTerm}
-          placeholder={searchPlaceholder}
-          onChange={this.onSearchTermChange}
-          className={styles.input}
-          endAdornment={
-            <InputAdornment position="end">
-              <Icon>{searchIcon}</Icon>
-            </InputAdornment>
-          }
-        />
-      </FormControl>
-    );
+      const SearchRenderer = this.props.searchRenderer;
+      return (
```

Comment:
Do we need this method? we can simply do this in the renderer right?

Reply:
right!